src/CubedSpheres/conformal_cubed_sphere_grid.jl

```diff
@@ -127,28 +127,28 @@ function ConformalCubedSphereGrid(arch = CPU(), FT=Float64; face_size, z, face_h
 
     size, halo = face_size, face_halo
 
-    # +z face (face 1)
-    z⁺_face_grid = OrthogonalSphericalShellGrid(arch, FT; size, z, halo, radius, rotation=nothing)
+    # +x face (face 1)
```

Comment:
We need a sketch with lat-lon lines overlaying the cube so we understand the "+z", etc terminology!

Reply:
Yeap, I plan to add some figures in the docs. For now, is the figures at https://clima.github.io/CubedSphere.jl/stable/conformal_cubed_sphere/ helpful? +z is the face that looks in the positive z direction.